src/disp_s1/product_info.py

```diff
@@ -19,13 +19,13 @@ class DispProductInfo:
     attrs: dict
 
     @classmethod
-    def unwrapped_phase(cls):
-        """Return container of unwrapped phase specific information."""
+    def range_change(cls):
+        """Return container of range change specific information."""
         return cls(
-            name="unwrapped_phase",
-            description="Unwrapped phase",
+            name="range_change",
+            description="Range change",
```

Comment:
We should probably have a description that, at least, contains the sign convention- "Positive values and indicate apparent motion towards the platform."
might also be worth having language thats it's deformation plus noise? maybe your "range change" name covers that alright

Reply:
I think we should use the name `displacement` instead of `range_change` because of the sign convention.

Option-1:
```
float displacement(y, x) ;
	displacement:_FillValue = NaNf ;
	string displacement:units = "meters" ;
	string displacement:long_name = "Displacement with noise in LOS. Positive values indicate apparent motion towards the platform." ;
	string displacement:grid_mapping = "spatial_ref" ;
```


Option-2:
```
float displacement(y, x) ;
	displacement:_FillValue = NaNf ;
	string displacement:units = "meters" ;
	string displacement:long_name = "Displacement (w/noise) in LOS. Positive values indicate apparent motion towards the platform." ;
	string displacement:grid_mapping = "spatial_ref" ;
```

any preferences from these options or other recommendations?